Source/KSCrash/Recording/Tools/KSLogger.h

```diff
@@ -260,6 +260,14 @@ bool kslog_setLogFilename(const char* filename, bool overwrite);
 /** Clear the log file. */
 bool kslog_clearLogFile();
 
+/** Set whether or not to log to stdout.
+ *
+ * @param enabled Whether or not stdout logging should be enabled.
+ *
+ * By default stdout logging is enabled.
+ */
+void kslog_setLogToStdout(bool enabled);
```

Comment:
> KSCrash already stores a fresh logfile to disk on every run.

Right, using `kslog_setLogFilename`, which I though we determined should not be called from KSCrash itself, unless it's passing along an explicit user request.

So to keep those two features we need to teach the logging framework to log to multiple files:
 - the internal log file used by `kscrash_setAddConsoleLogToReport`
 - whatever is set by the user via `kslog_setLogFilename`

Unless we make KSCrash set a default `kslog_setLogFilename` (and leave stdout enabled), which can be overridden by the user calling `kslog_setLogFilename` which would then disable stdout, and then make the `kscrash_setAddConsoleLogToReport` feature use whatever filename was set (by the user or the default), not rely on reading the log from ConsoleLog.txt?